README.md

```diff
@@ -78,9 +78,9 @@ Currently, compilation only targets Windows systems and requires Visual Studio
    aforementioned location called *build*.
 4. Check the **Grouped** and **Advanced** options.
 5. Press **Configure**. Select *Visual Studio 15 2017* as the generator.
-6. Set the *CMAKE_INSTALL_PREFIX* option under CMAKE to the *cwe-prefix* folder
-   that you extracted from the [development libraries
-   bundle](https://github.com/H-uru/PlasmaPrefix/releases/download/2020.05.01/devlibs.zip).
+6. Set the *CMAKE_INSTALL_PREFIX* option under CMAKE to the location that you
```

Comment:
The phrasing "location that you extracted" sounds fine spoken, but "location where you extracted" might be clearer in written form.

Reply:
Fixed